ooniprobe/Model/NetworkMeasurement.mm

```diff
@@ -16,11 +12,17 @@
 #include <resolv.h>
 #include <dns.h>
 
+#define verbosity MK_LOG_INFO
+#define anomaly_green 0
+#define anomaly_orange 1
+#define anomaly_red 2
```

Comment:
It's generally common practice to define constants using `CAPTAIN_CAPS_LOCK_CASE`